stages/org.osbuild.kickstart

```diff
@@ -342,6 +346,9 @@ def main(tree, options):
     clearpart = make_clearpart(options)
     if clearpart:
         config += [clearpart]
+    text = options.get("text")
```

Comment:
The assignment here is unnecessary as `text` only gets used one line down. I don't care particularly since the idiom is consistent with all the lines above it but if you *want* to change it you can :)